### PR TITLE
[multistage] add test for LiteralValueOperator

### PR DESCRIPTION
**Pull request by agavra**
Super quick and easy test :)

### CODE REVIEW

**Review by siddharthteotia — APPROVED** [review]
May be we also want to add a simple test for input null / empty and the operator shouldn't NPE or anything, return empty.

File: pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/operator/LiteralValueOperatorTest.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    );
+    LiteralValueOperator operator = new LiteralValueOperator(schema, literals);
+
+    // When:
+    TransferableBlock transferableBlock = operator.nextBlock();
+
+    // Then:
+    Assert.assertEquals(transferableBlock.getContainer().get(0), new Object[]{"foo", 1});
+    Assert.assertEquals(transferableBlock.getContainer().get(1), new Object[]{"bar", 2});
+    Assert.assertTrue(operator.nextBlock().isEndOfStreamBlock(), "Expected EOS after reading two rows");
```
Comment (walterddr) [inline pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/operator/LiteralValueOperatorTest.java:55]: can you also add a test for literals that are empty?

**Review by walterddr — APPROVED** [review]
lgtm. but also could you add test for empty literals?

File: pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/operator/LiteralValueOperatorTest.java (hunk 1 above)
Comment (agavra) [reply pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/operator/LiteralValueOperatorTest.java:55]: added, but how can you create an empty literal in first place (in SQL)?

Comment (walterddr) [reply pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/operator/LiteralValueOperatorTest.java:55]: `SELECT col1 FROM tbl WHERE col2 BETWEEN 0 AND -1` will generate an empty literal value node with schema of col1 and 0 rows.
